terraform/terraform-create-resource/cosmosdb.tf

```diff
@@ -0,0 +1,43 @@
+# azurerm_cosmosdb_account.db:
+resource "azurerm_cosmosdb_account" "db" {
+    access_key_metadata_writes_enabled    = true
+    analytical_storage_enabled            = false
+    enable_automatic_failover             = false
+    # Only one free tier cosmosDB per subscription
+    # enable_free_tier                      = true
+    enable_multiple_write_locations       = false
+    is_virtual_network_filter_enabled     = false
+    kind                                  = "GlobalDocumentDB"
+    local_authentication_disabled         = false
+    location                              = "southeastasia"
+    name                                  = "${var.prefix}-couple-management-storage"
+    network_acl_bypass_for_azure_services = false
+    network_acl_bypass_ids                = []
+    offer_type                            = "Standard"
+    public_network_access_enabled         = true
+    resource_group_name                   = azurerm_resource_group.rg.name
+
+    capabilities {
+    name = "EnableServerless"
```

Comment:
incorrect indentation I think.

Also, could document down what IDE you use to work with Terraform files (VS Code with some plugin installed?)